### ISSUE TITLE
Not able to download florplan image

### ISSUE THREAD

**Issue by satyampv** [IndoorAtlas/android-sdk-examples#2]
I tried to run the code into android 4.4.2.
I was not able to download a image floor plan.

**Comment by hongweig** [comment]
Check that you have configured IndoorAtlas API key, and once you have API keys, edit them into gradle.properties in the project root level.

**Comment by satyampv** [comment]
Hello All
Appreciated your quick feedback on the query !!

I tried to change the API keys & Security keys as well.

Instead of change into the Gradle system as well.
I put direct API keys & Security into AndroidManifest file.
I want to download only floor map so Changed only this portion of the code.

Ref.
AnroidManifest.xml

<meta-data
    android:name="com.indooratlas.android.sdk.API_KEY"
    android:value="c8b1728e-8822-415b-a22f-e9a07b42f0a0"/>
<meta-data
    android:name="com.indooratlas.android.sdk.API_SECRET"
    android:value="Zaxv0eA9sn%Lf5UGRgx5zd3fWVVgk8W4rqp90smoARNbpuiauHBy8L4(qBscQVT46lgfWlKwLr!IA2IU!yLHRb6KIAc%r1IMK34IziMBaTCcRcWUYU96Eu6I)Ai)9KtQ"/>

gradle.Properties


// Edit in your IndoorAtlas SDK API credentials here
indoorAtlasApiKey=c8b1728e-8822-415b-a22f-e9a07b42f0a0
indoorAtlasApiSecret=Zaxv0eA9sn%Lf5UGRgx5zd3fWVVgk8W4rqp90smoARNbpuiauHBy8L4(qBscQVT46lgfWlKwLr!IA2IU!yLHRb6KIAc%r1IMK34IziMBaTCcRcWUYU96Eu6I)Ai)9KtQ

I removed that public APIS call.

Nothing about this is changed from the code.
Should it device issue ?

I am using device :
device::product* = 'ja3gxx*
12-10 15:53:07.106 11704-11875/com.indooratlas.android.sdk.examples
I/indooratlas_cpa: *device::systemName = 'Android'*
12-10 15:53:07.106 11704-11875/com.indooratlas.android.sdk.examples
I/indooratlas_cpa: *device::systemVersion = '4.4.2*
12-10 15:53:07.106 11704-11875/com.indooratlas.android.sdk.examples
I/indooratlas_cpa: '
Samsung s4

Regards
Satyam



On Thu, Dec 10, 2015 at 3:38 PM, Hongwei <notifications@github.com> wrote:

> Check that you have configured IndoorAtlas API key, and once you have API
> keys, edit them into gradle.properties in the project root level.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IndoorAtlas/android-sdk-examples/issues/2#issuecomment-163563538>
> .
>



-- 
Satyam Vakharia

+91 9824839928
Gtalk: spvakharia3@gmail.com
skype: spvakharia
